features/collections/collection_notification.feature

```diff
@@ -64,6 +64,67 @@ Feature: Collectible items email
       And I press "Create"
     Then 1 email should be delivered
 
+  Scenario: Scenario: Translated email is sent when the status of a Collection item is changed to anonymous
```

Comment:
I think something got tangled here: The tests still has the name with the extra "Scenario" and then there is a duplicated test down below. Same for the other two tests.